### ISSUE TITLE
problem in test in talkset

### ISSUE THREAD

**Issue by coreeey** [TaoRuijie/TalkNet_ASD#36]
Hello, I generated the talkset according to generate_talkset, used the talkset pre-training model provided by you, tested the model on the validation set of talkset, but i found, in my code,  the mAP is only 55%, and i am very confused about it. I set the TAudio label to 1, and the rest to 0, may I ask you is it set up in this way? Is it convenient to expose the talkset Dataset?

**Comment by TaoRuijie** [comment]
You can check this:https://github.com/TaoRuijie/TalkNet-ASD/issues/4#issuecomment-889714345 I provide the dataloader, it contains the info about the label.

I think the lable is incorrect, TFAudio and TSilence contain a component with the label 1. You can check https://github.com/TaoRuijie/TalkNet-ASD/blob/main/TalkSet/README.md

**Comment by coreeey** [comment]
Thank u, i find my label is incorrect
